### ISSUE TITLE
Centralising notes from the accessibility working group

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/the-turing-way#3075]
### Summary

# Accessibility Guidance for Events (Hybrid or in-person)

:::success

:information_source: How to work on this document.

This is a **collaborative document** where multiple people can **add, review or update best practices** to offer **accessible events**. 

This document aims to propose guidelines to prepare accessible events, either online or in-person. 

:::

### Accessibility

Accessibility and inclusion should be a key consideration in any event design. Participants with visible or invisible disabilities should be welcome and materials should be available in the appropriate formats to make them accessible to all. 

### Best Practices for hosting inclusive  conferences

1. **Event design**: 
- Clearly emphasize in all your event planning documents and tasks the importance of accessibility and inclusion. 
- Make it clear to speakers, suppliers and attendants and provide them with relevant materials to use/follow during the event.
- For international events: Consider scheduling activities in multiple time zones so that everyone in the world can watch at least part of the conference live, and provide a link to all recorded sessions so people can catch up on what they missed   

2. **In-person Events**: 
- Make sure that the venue has accessible entrance, corridors, toilets, lifts for wheelchairs
- Be mindful of sensory needs, provide quiet spaces, and make attendants aware if there will be loud noises, videos, etc. 
- The sound system at the venue should have the audio induction loop system that can send signals directly to hearing aids. The availability of this system should be posted on a sign at the entrance to the room, along with specifications if only a certain part of the room is covered.
- At meals and breaks, someone should be available to help blind and low vision people to navigate food and beverage choices. 
- Provide a wide range of foods including gluten- and allergen-free, as well as vegetarian/vegan, options. Make ingredient lists available
3. **Online and hybrid events**: 
- Test the internet, cameras, light and have a backup plan in case the connection of the main host fails. Assign more than one host, two co-host.
- Make materials available offline (slides, code and records) and allow the audience to access your materials before, during and after the conference.
- Add captions to your video or provide a transcript of your talk so it’s accessible to people who are blind, deaf, hard-of-hearing, or have low-vision and may be using software such as screen readers. Always keep cations on. There may be times when you don't want to be recording, but people who are present  should still be able to benefit from the captions during that time.
- Record the presentations and share them after the event. 
- For hybrid events, encourage attendants to follow the conversation by email, or other platforms that allow interaction between people also attending online. 

4. **Presentation slides, materials and images**: 
- Use an accessible slide format. For example, markdown with alt text is more screen reader friendly than PowerPoint, and PDFs of slides are often very difficult to read
- Make sure all materials have legible text with distinguishable font and font size, so people can follow the conference from a small screen. For example, use Arial at least a 28 point size. Also, resize images so they can be viewed even on a small screen.
- Make materials available prior to the event 
- Do a control review of all presentations to make sure they are accessible. *(Ask Liz which systems she recommends)*
- Do not forget to include alt-text to all your figures and images

5. **Presenters and Speakers**: 
- During your talk, try to speak as clearly as possible and leave your mouth visible to allow people to read your lips if necessary
- When possible, sign language and/or simoultaneuos translation should be included
- Presenters and speakers should use accessible language and graphics, pictures, videos, and memes should be described audibly, if they don't have Alt text
- To help presenters, the organisers should provide a template that covers all our technical accessibility considerations. Templates To be done.

6. **Feedback**: 
- Always include a set of questions about accessibility in your feedback forms. 
- Any recommendations or new practices, should be added here. 

### Next actions
1. Finish the guide for accessibility at The Turing Way book : https://github.com/alan-turing-institute/the-turing-way/blob/accessibility-guide/book/website/accessibility-guide/accessibility-guide.md
2. Develop a Checklist
3. Develop templates

#### References:
1. [Turing Institute: Equality, diversity and inclusion strategy 2021-2024](https://www.turing.ac.uk/sites/default/files/2022-01/edi-strategy-report_v1.2_accessible_updated_02.12.2021.pdf#:~:text=The%20Institute%20has%20defined%20a%20set%20of%20values,values%3A%20trust%2C%20inclusivity%2C%20respect%2C%20leadership%2C%20transparency%2C%20and%20integrity).
2. [R Use - Preparing for an accesible online conference](https://user2021.r-project.org/blog/2021/02/17/preparing-for-an-accessible-conference/)
3. [R use - Making accessible presentations at user! 2021: The story behind the scenes](https://user2021.r-project.org/blog/2021/12/07/accessibility_awards_interview/)
4. [Ten simple rules to host an inclusive conference](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010164)

**Comment by malvikasharan** [comment]
### Accessibility

Accessibility and inclusion should be a key consideration in any event design. Participants with visible or invisible disabilities should be welcome and materials should be available in the appropriate formats to make them accessible to all. 

### Best Practices for hosting inclusive  conferences

1. **Event design**: 
- Clearly emphasize in all your event planning documents and tasks the importance of accessibility and inclusion. 
- Make it clear to speakers, suppliers and attendants and provide them with relevant materials to use/follow during the event.
- For international events: Consider scheduling activities in multiple time zones so that everyone in the world can watch at least part of the conference live, and provide a link to all recorded sessions so people can catch up on what they missed   

2. **In-person Events**: 
- Make sure that the venue has accessible entrance, corridors, toilets, lifts for wheelchairs
- Be mindful of sensory needs, provide quiet spaces, and make attendants aware if there will be loud noises, videos, etc. 
- The sound system at the venue should have the audio induction loop system that can send signals directly to hearing aids. The availability of this system should be posted on a sign at the entrance to the room, along with specifications if only a certain part of the room is covered.
- At meals and breaks, someone should be available to help blind and low vision people to navigate food and beverage choices. 
- Provide a wide range of foods including gluten- and allergen-free, as well as vegetarian/vegan, options. Make ingredient lists available
3. **Online and hybrid events**: 
- Test the internet, cameras, light and have a backup plan in case the connection of the main host fails. Assign more than one host, two co-host.
- Make materials available offline (slides, code and records) and allow the audience to access your materials before, during and after the conference.
- Add captions to your video or provide a transcript of your talk so it’s accessible to people who are blind, deaf, hard-of-hearing, or have low-vision and may be using software such as screen readers. Always keep cations on. There may be times when you don't want to be recording, but people who are present  should still be able to benefit from the captions during that time.
- Record the presentations and share them after the event. 
- For hybrid events, encourage attendants to follow the conversation by email, or other platforms that allow interaction between people also attending online. 

4. **Presentation slides, materials and images**: 
- Use an accessible slide format. For example, markdown with alt text is more screen reader friendly than PowerPoint, and PDFs of slides are often very difficult to read
- Make sure all materials have legible text with distinguishable font and font size, so people can follow the conference from a small screen. For example, use Arial at least a 28 point size. Also, resize images so they can be viewed even on a small screen.
- Make materials available prior to the event 
- Do a control review of all presentations to make sure they are accessible. *(Ask Liz which systems she recommends)*
- Do not forget to include alt-text to all your figures and images

5. **Presenters and Speakers**: 
- During your talk, try to speak as clearly as possible and leave your mouth visible to allow people to read your lips if necessary
- When possible, sign language and/or simoultaneuos translation should be included
- Presenters and speakers should use accessible language and graphics, pictures, videos, and memes should be described audibly, if they don't have Alt text
- To help presenters, the organisers should provide a template that covers all our technical accessibility considerations. Templates To be done.

6. **Feedback**: 
- Always include a set of questions about accessibility in your feedback forms. 
- Any recommendations or new practices, should be added here. 

### Next actions
1. Finish the guide for accessibility at The Turing Way book : https://github.com/alan-turing-institute/the-turing-way/blob/accessibility-guide/book/website/accessibility-guide/accessibility-guide.md
2. Develop a Checklist
3. Develop templates



#### References:
1. [Turing Institute: Equality, diversity and inclusion strategy 2021-2024](https://www.turing.ac.uk/sites/default/files/2022-01/edi-strategy-report_v1.2_accessible_updated_02.12.2021.pdf#:~:text=The%20Institute%20has%20defined%20a%20set%20of%20values,values%3A%20trust%2C%20inclusivity%2C%20respect%2C%20leadership%2C%20transparency%2C%20and%20integrity).
2. [R Use - Preparing for an accesible online conference](https://user2021.r-project.org/blog/2021/02/17/preparing-for-an-accessible-conference/)
3. [R use - Making accessible presentations at user! 2021: The story behind the scenes](https://user2021.r-project.org/blog/2021/12/07/accessibility_awards_interview/)
4. [Ten simple rules to host an inclusive conference](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010164)
